src/Microsoft.ML.DnnImageFeaturizer.ResNet50/ResNet50.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.ML.Transforms
+{
+    /// <summary>
+    /// This is an extension method to be used with the <see cref="DnnImageFeaturizerEstimator"/> in order to use a pretrained ResNet50 model.
+    /// The NuGet containing this extension is also guaranteed to include the binary model file. Note that when building the project
+    /// containing this extension method, the corresponding binary model will be downloaded from the CDN at
+    /// https://express-tlcresources.azureedge.net/image/ResNetPrepOnnx/ResNetPreprocess.onnx and
```

Comment:
have we not renamed `tlcresources` yet?  If not, we should push to get it renamed/aliased so we don't have to keep putting `TLC` things in the code.

Reply:
We have not, but I have changed this to use the aka.ms aliased URL instead which does not contain it.

---
In reply to: [232770927](https://github.com/dotnet/machinelearning/pull/1447#discussion_r232770927) [](ancestors = 232770927)